### ISSUE TITLE
Worker/offscreen canvas example

### ISSUE THREAD

**Issue by lwansbrough** [gfx-rs/wgpu#1466]
Would be nice if there was a (probably broken) demo that utilized a worker for offscreen canvas rendering, sort of as a target to hit. I imagine the APIs will change, but having the demo there could serve as way for those of us interested to check on the status of that specific aspect of the spec.

**Comment by cwfitzgerald** [comment]
Just as a point of clarity: this is about rendering to an offscreen canvas not render-to-texture?

**Comment by lwansbrough** [comment]
> Just as a point of clarity: this is about rendering to an offscreen canvas not render-to-texture?

Correct, a canvas whose control has been transferred to a web worker.
